release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -44,6 +44,20 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-12-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-12-default-graphical-console"]
+==== Default consoles for new clusters are now determined by the installation platform
+{op-system-first} nodes installed from an {product-title} {product-version} boot image now use a platform-specific default console. The default consoles on cloud platforms correspond to the specifc system consoles expected by that cloud provider. VMware images and bare metal installations now use only the graphical console by default, and do not enable a serial console. Installations performed with `coreos-installer` can override existing defaults and enable the serial console.
```

Comment:
Typo:

```suggestion
{op-system-first} nodes installed from an {product-title} {product-version} boot image now use a platform-specific default console. The default consoles on cloud platforms correspond to the specific system consoles expected by that cloud provider. VMware images and bare metal installations now use only the graphical console by default, and do not enable a serial console. Installations performed with `coreos-installer` can override existing defaults and enable the serial console.
```

Also, VMware and OpenStack images have now been changed to use a primary graphical and secondary serial console so that the serial console can still be used for debugging.  (Previously, both used a primary serial and secondary graphical console.  The difference between primary and secondary is that some messages only go to the primary console.)